pandas/core/indexes/base.py

```diff
@@ -1113,6 +1113,26 @@ def _format_attrs(self):
         """
         return format_object_attrs(self)
 
+    def to_flat_index(self):
+        """
+        Identity method.
+
+        .. versionadded:: 0.24.0
+
+        This is implemented for compatability with subclass implementations
```

Comment:
compatability -> compatibility

By the way: reusing the same docstring would be a + for me. I would rather describe the actual operation (so on ``MuiltiIndex``) and then say that it is idempotent on flat ``Index``es.